src/Icons/doc/index.rst

```diff
@@ -526,13 +526,13 @@ TwigComponent
 ~~~~~~~~~~~~~
 
 The ``ux_icon()`` function is optimized to be as fast as possible. To deliver the
-same level of performance when using the HTML syntax (``<twig:ux:icon name="..." />``),
+same level of performance as if using the HTML syntax (``<twig:ux:icon name="..." />``),
 the TwigComponent overhead is reduced by calling the IconRenderer immediately and
 returning the HTML output.
 
 .. warning::
 
-    The <twig:ux:icon> component does not support embedded content.
+    The ``<twig:ux:icon>`` tag does not support embedded content.
```

Comment:
No. I'm sorry. It's not a question of tag but a question of special Twig Component